index.js

```diff
@@ -52,17 +52,18 @@ client.on('message', async msg => {
 			\n\bWelcome to the eSSE's mentoring system! We're here to help.\
 			\n\bHere's a few helpful commands:\
 			```\
-			\n\b!help -> See this command (but you knew that already)\
+			\b!help -> See this command (but you knew that already)\
 			\n\b!ping -> Make mentors aware you need help (Please use discretion, there may only be one mentor online at a time)\
 			\n\b!join -> Enters you in a voice channel\
 			```\
 			\n\bPlease remember the following items:\
 			```\
-			\n\bWe are volunteers (we don't get paid)\
+			\bWe are volunteers (we don't get paid)\
 			\n\bOur hours are 10 am - 6 pm\
 			\n\bWe are an official RIT organization. PLEASE no profanity, harrassment, sexual comments, or anything else made to mentors or other mentees.\
 			\n\bIf you are overly aggressive to our mentors or break any of the rules above, you will be banned permanently and you will be reported to RIT\
 			```\
+			\n\b*If you would like to mute this channel to prevent being spammed with notifications, right click on the channel in the navigation bar to the left, navigate to \"Notifications\" and select the option most applicable to your needs.*\
```

Comment:
Updated the instructions to include options to still receive notifications

Reply:
Thought: should we have the Bot/Mentor send a notification to the mentee when they join their channel? So they are notified when someone is there to help

Reply:
So we should tell them to block all but @mentions

Reply:
Nevermind, dumb idea cause they will get their own channel that won't be muted when they request one

Reply:
I'm picking up what you're putting down now... I went through and changed to still receive @mentions